spec/cart_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "cart"
+
+
+
+describe "Order" do 
```

Comment:
I really like the layout for your testing methods - they're super tidy and really easy to read!